docker-compose.yml

```diff
@@ -15,7 +15,7 @@ secrets:
 
 services:
   mailer:
-    image: cisagov/cyhy-mailer:1.6.0
+    image: cisagov/cyhy-mailer:1.7.1
```

Comment:
```suggestion
    image: cisagov/cyhy-mailer:1.7.0
```

Reply:
I cleaned this PR up and force-pushed https://github.com/cisagov/cyhy-mailer/pull/108/commits/25702fa363a046f51cd69982234fbdee3f2ec3e0.  Please let me know if you have any other issues here.